connector_helpscout/models/helpscout_backend/common.py

```diff
@@ -24,12 +24,15 @@ class HelpscoutBackend(models.Model):
     _inherit = 'connector.backend'
 
     EVENT_TO_MODEL = {
+        'attachment': 'helpscout.attachment',
```

Comment:
This is meant to map web hook prefixes to models they reference. There aren't `attachment`, `tag`, and `thread` hooks though - what are you using this for?

Reply:
Whoops. I didn't look up the events, just kept adding as I went.